### PR TITLE
bijan/TradePage_Validations

### PR DESCRIPTION
**Pull request by bijan2007**
1. Check all markets are displayed in the menu
2. Check all underlying assets are displayed for selected market
3. Check error validation for duration fields
4. Check error validation for payout field
5. Check error validations for barrier field

### CODE REVIEW

File: src/test/java/appModules/Trading_Action.java [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+			Assert.assertEquals(Trade_Page.err_Payout5000Top(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			Assert.assertEquals(Trade_Page.err_Payout5000Bottom(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+		}
+		else if (market=="Commodities"){
+			SelectEnterAmount(driver,"0.49",amount_type,"5","m");
+			Assert.assertEquals(Trade_Page.err_Payout5000Top(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			Assert.assertEquals(Trade_Page.err_Payout5000Bottom(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			SelectEnterAmount(driver,"5100",amount_type,"5","m");
+			Assert.assertEquals(Trade_Page.err_Payout5000Top(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			Assert.assertEquals(Trade_Page.err_Payout5000Bottom(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
```
Comment (ridhomard) [inline src/test/java/appModules/Trading_Action.java]: I got error 

> java.lang.AssertionError: expected [Minimum stake of 0.50 and maximum payout of 5,000.00] but found [Minimum stake of 0.50 and maximum payout of 5,000.00.]

when ran in my local , could you help to check.

**Comment by ridhomard** [comment]
Overall it seems good - covers most of the validation but there is some suggestion. 

- We should check absolute barrier validation 
> Contracts more than 24 hours in duration would need an absolute barrier.

- We need to check the correct behaviour of trading page (not only validation). Like when all parameters is valid client should be able to buy (detecting by a buy button ?)

File: src/test/java/appModules/Trading_Action.java (hunk 1 above)
Comment (bijan2007) [reply src/test/java/appModules/Trading_Action.java]: Yes. Seems there were front-end changes to error message in recent release. I've fixed the issue.

**Comment by bijan2007** [comment]
@ridhomard These tests already cover absolute barrier. Are you talking about High/Low barrier ?. Can you send me across some screenshot. 
Also regarding correct behaviour of trading page I will cover it in another PR.

**Comment by ridhomard** [comment]
Yes higher/lower - when you + in the barrier for daily contract - i will send you through slack

**Review by ridhomard — APPROVED** [review]

**Pull request merged**